install/roles/flavors/vars/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+pbench_flavors:
+  - name: pbench_flavor
```

Comment:
Why have flavors named after "pbench" instead of the workloads you want to have pbench wrap?

Reply:
As the flavor is used to spawn uperf client vm and uper server vm for testing